jerry-ext/CMakeLists.txt

```diff
@@ -20,11 +20,12 @@ project (${JERRY_EXT_NAME} C)
 set(INCLUDE_EXT "${CMAKE_CURRENT_SOURCE_DIR}/include")
 
 # Source directories
-file(GLOB SOURCE_EXT arg/*.c)
+file(GLOB SOURCE_EXT arg/*.c context/*.c)
 
 add_library(${JERRY_EXT_NAME} STATIC ${SOURCE_EXT})
 
 target_include_directories(${JERRY_EXT_NAME} PUBLIC ${INCLUDE_EXT})
+target_include_directories(${JERRY_EXT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/private-headers")
```

Comment:
Sorry for being unclear! I wish for this file to remain in a directory that can be referenced from multiple subdirectories of `jerry-ext/` because both `jerry-ext/context/context.c` and the future `jerryx-ext/module/module.c` will `#include` it.